app/sidekiq/evss/disability_compensation_form/submit_form526.rb

```diff
@@ -175,6 +175,7 @@ def retryable_error_handler(_submission, error)
       def non_retryable_error_handler(submission, error)
         # update JobStatus, log and metrics in JobStatus#non_retryable_error_handler
         super(error)
+        submission.mark_birls_id_as_tried!
         submission.submit_with_birls_id_that_hasnt_been_tried_yet!(
```

Comment:
if we call `submission.mark_birls_id_as_tried!`, wouldn't it follow that we'd always be calling `submission.submit_with_birls_id_that_hasnt_been_tried_yet!` with absolutely nothing? 😂 Or am I placing too much trust on how these methods are named?

Reply:
Updates made, @sethdarragile6, per your feedback. Re-review appreciated.